students/ivan/carrito-compra/script.js

```diff
@@ -0,0 +1,255 @@
+/*
+1-Crea una clase de tipo producto que contenga nombre, imagen  y precio.
+*/
+class Product {
+
+    constructor(name,img,price){
+
+        this.name = name;
+        this.image = img;
+        this.price = price;
+    }
+}
+
+/*
+2- luego crea varios productos de la clase Producto.
+voy a crear 5
+*/
+
+const bike = new Product ('Bike','url(./src/bike.jpg)',250);

```

Comment:
we usually use URL for image instead og background image value. For example instead o `url(./src/bike.jpg)` we shoukld use `./src/bike.jpg` so we can also use it in html img tag